rf_tests.go

```diff
@@ -362,6 +369,108 @@ func uploadRFML(c cliContext, api rfmlAPI) error {
 	if err != nil {
 		return cli.NewExitError(err.Error(), 1)
 	}
+
+	wisp_tests, err := readWispFiles([]string{c.String("test-folder")})
+	if err != nil {
+		return cli.NewExitError(err.Error(), 1)
+	}
+	err = uploadWispFiles(wisp_tests, false, api)
+	if err != nil {
+		return cli.NewExitError(err.Error(), 1)
+	}
+
+	return nil
+}
+
+// uploadSingleWispFile uploads wisp file syntax by
+// trying to parse the file and sending any parse errors to the caller
+func uploadSingleWispFile(filePath string) error {
+	wisp, _ := readWispFile(filePath)
+
+	log.Printf("Updating existing wisp test: %v", wisp.TestID)
+	err := api.UpdateWisp(wisp)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func readWispFile(filePath string) (*rainforest.WispJson, error) {
+	file, err := os.Open(filePath)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+
+	wispReader := rainforest.NewWispReader(file)
+	wisp, err := wispReader.Read()
+	if err != nil {
+		return nil, fileParseError{filePath, err}
+	}
+
+	return wisp, err
+}
+
+// readWispFiles takes in a list of files and/or directories and a list of tags
```

Comment:
What are tags - copypasta from some other comment?

Reply:
Yes, I'll remove it. It doesn't work with wisp at least atm, not sure if that's something we'll add to match RFML.